pygac/lac_reader.py

```diff
@@ -48,11 +48,10 @@ def _validate_header(self):
         # call super to enter the Method Resolution Order (MRO)
         super(LACReader, self)._validate_header()
         LOG.debug("validate header")
-        data_set_name = self.head['data_set_name']
+        data_set_name = self.head['data_set_name'].decode()
         # split header into parts
-        # TODO: use trollshift
-        creation_site, transfer_mode, platform_id, _ = (
-            data_set_name.decode().split('.', maxsplit=3)
+        creation_site, transfer_mode, _ = (
+            data_set_name.split('.', maxsplit=2)
         )
-        if transfer_mode == 'GHRR':
-            raise ReaderError('Wrong transfer mode "%s"!' % transfer_mode)
+        if transfer_mode != 'LHRR':
```

Comment:
IIRC it can also be 'HRPT'

Reply:
From the [KLM guide](https://www1.ncdc.noaa.gov/pub/data/satellite/publications/podguides/N-15%20thru%20N-19/pdf/2.5%20Section%208.0%20NOAA%20Level%201B%20Database.pdf) I got the following information:
```
Four characters identifying the data type and transmission method.
Recognized character groups are:
HRPT= HRPT (direct readout full resolution AVHRR)
GHRR= GAC (recorded reduced resolution AVHRR)
LHRR= LAC (recorded HRPT AVHRR)
FRAC=FRAC (recorded full resolution, full orbit data from Metop)
HIRX= HIRS/3 or HIRS/4 data set derived from GAC embedded TIP
AMAX= AMSU-A data set derived from GAC embedded TIP
AMBX= AMSU-B data set derived from GAC embedded TIP
MHSX=MHS data set derived from AIP
MHSS=MHS data derived from stored AIP
MHSH=MHS data derived from HRPT
HIRS= HIRS/3 or HIRS/4 data set derived from stored TIP
AMAS= AMSU-A data set derived from stored TIP
AMBS= AMSU-B data set derived from stored TIP
```
Therefore, I thought that GAC only allows GHRR, but please let me know what else can be recognized as GAC-file. Same holds for the other `XxxReader._validate` methods, please have a look at them to check if the selection rule is set correctly.

Reply:
We tried with HRPT and LHRR and both worked with the LAC reader. FRAC should work too, but I think @helgaweb didn't succeed reading it.

Reply:
Never tried to read FRAC, however, FRAC can be 1km or 4km.
'AVHRR FRAC data, the user may choose 1 km or 4 km as their default, available in swaths typically covering roughly an orbits worth of data.' for Metop Global Data Services
So, which is the resolution we are talking about? Then it might be LHRR or GHRR?

Reply:
I was thinking 1km. I though the metop data you had was FRAC, sorry :)